client/components/FaveList.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import FaveListItem from "./FaveListItem";
+import { FaveWithDetails, ZoopWithDetails } from "../../src/types/custom";
+
+import Stack from "@mui/material/Stack";
+
+type FaveListProps = {
+  zoop: ZoopWithDetails;
+};
```

Comment:
Nit: newline between prop types and component
```suggestion
};

```